code/controllers/subsystems/initialization/codex.dm

```diff
@@ -0,0 +1,121 @@
+
+SUBSYSTEM_DEF(codex)
+	name = "Codex"
+	flags = SS_NO_FIRE
+	init_order = SS_INIT_CODEX
+
+	/// List of cooking recipes, their result and ingredients.
+	var/list/cooking_codex_data = list()
+
+	var/list/chemistry_codex_data = list()
+	var/list/chemistry_codex_ignored_reaction_path = list(/datum/chemical_reaction/slime)
+	var/list/chemistry_codex_ignored_result_path = list(/singleton/reagent/drink, /singleton/reagent/alcohol)
+
+/datum/controller/subsystem/codex/Initialize()
+	generate_cooking_codex()
+	generate_chemistry_codex()
+
+	LOG_DEBUG("SScodex: [cooking_codex_data.len] cooking recipes; [chemistry_codex_data.len] chemistry recipes.")
```

Comment:
This needs its own subsystem to log, if it needs to log anything; See a file in `code\__HELPERS\logging\subsystems\` for reference